packages/eas-json/src/log.ts

```diff
@@ -0,0 +1,16 @@
+import chalk from 'chalk';
```

Comment:
In the past, we tried to avoid printing logs from eas-json. Move error detection to eas-cli.